laws/views.py

```diff
@@ -965,7 +965,7 @@ def committee_meeting_auto_complete(request):
 
     optional_committee_meetings = CommitteeMeeting.objects.filter(
         Q(date_string__icontains=q) | Q(topics__icontains=q)
-    )
+    )[:10]
```

Comment:
But does't this remove meetings that **should** be available? maybe a better strategy would be using topics only when q is a topic, or prefetch_related topics or try to give more information for the q so the basic query would have less results and be more precise